### DISCUSSION TITLE
Change node.url based on current language (Jekyll Multiple Language Plugin)

### DISCUSSION THREAD

**Discussion by zkvvoob** [jekyll/jekyll#9133] [Q&A]
Hello,

I'm trying to set up a website based on Jekyll, a theme called [Just the Docs](https://github.com/just-the-docs/just-the-docs) and the Jekyll Multiple Languages Plugin. So far I've been able to make the individual pages' titles display based on the selected language. However, I now face a new challenge, namely - how to set the proper URLs to point to the `/en/` subdirectory.

In `nav.html` I'm looking at this string

```html
<a href="{{ node.url | absolute_url }}" class="nav-list-link{% if page.url == node.url %} active{% endif %}">{% t node.title %}</a>
```

*Note how I've changed `{{node.title}}` to `{% t note.title %}` and I can now get the respective language's translated title.*

I'm thinking something along these lines:

```html
{% if site.lang == "bg" %}
    <a href="{{ node.url | absolute_url }}" class="nav-list-link{% if page.url == node.url %} active{% endif %}">{% t node.title %}</a>
{% elsif site.lang == "en" %}
    <a href="/en{{ node.url | absolute_url }}" class="nav-list-link{% if page.url == node.url %} active{% endif %}">{% t node.title %}</a>
{% endif %}
```

However, that only duplicates the whole URL after `/en/`, i.e. `http://localhost:4000/en/http://localhost:4000/publication-title/`.

So, could someone help me figure out the correct variable to use in order to accomplish something like this:
- if site.lang == "bg" then original node.url
- else siteURL/en/{pageURL}

Thank you!

**Comment by pranabdas** [comment]
What happens if you use `relative_url` instead of `absolute_url`?
